esdb/errors_test.go

```diff
@@ -0,0 +1,13 @@
+package esdb
+
+import (
+	"fmt"
+	"github.com/stretchr/testify/assert"
+	"testing"
+)
+
+func TestErrorIsErrorCode(t *testing.T) {
```

Comment:
Your test is currently not called during CI. You should add your `TestErrorIsErrorCode` below that line: https://github.com/EventStore/EventStore-Client-Go/blob/master/esdb/client_test.go#L91

Reply:
the `client_test.go` doesn't have access to the internals of the package; the reason why I added the new file is I couldn't find a way to set the `code` or `err` of the `Error` since they are internal properties.

Reply:
I think it's safe for you to remove that test entirely.

Reply:
✅